src/awsexplorer/regionNode.ts

```diff
@@ -82,6 +84,10 @@ const serviceCandidates: ServiceNode[] = [
         serviceId: 'ecs',
         createFn: (regionCode: string) => new TreeShim(getEcsRootNode(regionCode)),
     },
+    {
+        serviceId: 'emr-serverless',
+        createFn: (regionCode: string) => new EmrServerlessNode(new EmrServerlessClient(regionCode)),
+    },
```

Comment:
should probably have a `when` field which excludes this feature from cloud9, unless you tested in cloud9